src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -118,8 +118,9 @@ public enum ProtectionLevel
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public static readonly System.Net.Security.SslApplicationProtocol Http11;
+        public static readonly System.Net.Security.SslApplicationProtocol Http3;
         public static readonly System.Net.Security.SslApplicationProtocol Http2;
+        public static readonly System.Net.Security.SslApplicationProtocol Http11;
```

Comment:
Why did the ordering here change?

Reply:
I changed it to be consistent. In all other places the order was descending.
Should I keep it ascending?

Reply:
Where is the order descending?

We should keep this as ascending.

Reply:
Changed to ascending